packages/components/src/components/popover/popover.jsx

```diff
@@ -89,6 +90,9 @@ class Popover extends React.PureComponent {
                         transitionDuration={0.25}
                         padding={margin + 8}
                         contentDestination={relative_render ? this.state.popover_ref : document.body}
+                        containerClassName={classNames({
+                            'react-tiny-popover-container__disabled-pointer-event': should_disable_pointer_events,
```

Comment:
```suggestion
                            'react-tiny-popover-container__pointer-event--disabled': should_disable_pointer_events,
```
We use the BEM structure for naming classes. 
Block__Element--Modifier 
http://getbem.com/naming/